### PR TITLE
Add parameter outputImageName to method createResizedImage

### PR DESCRIPTION
**Pull request by fcostaprojects**
I added the parameter outputImageName, on method createResizedImage to allow changing the name of the resized image.
Android only.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@ compressFormat | Can be either JPEG, PNG or WEBP (android only).
 quality | A number between 0 and 100. Used for the JPEG compression.
 rotation | Rotation to apply to the image, in degrees, for android. On iOS, rotation is limited (and rounded) to multiples of 90 degrees.
 outputPath | The resized image path. If null, resized image will be stored in cache folder. To set outputPath make sure to add option for rotation too (if no rotation is needed, just set it to 0).
+outputImageName | The new name of image redized. (if no need outputPath, just set to null). Android only.
```
Comment (4ian) [inline README.md]: there a small typo ;) Also I'm not sure about what you mean by  "if no need outputPath, just set to null"?
I think something like `outputFilename | The name of the output file (if not defined, a random name will be generated). Android only.`

**Comment by 4ian** [comment]
Thanks for the PR! 👍 
I'm wondering if `outputFilename` would be a better name compared to `outputImageName`? As we have already outputPath, it make sense to have outputFilename. ImageName is less specific than Filename, which convey the clear meaning that is the name of the file that is created :)
What do you think? Looks good otherwise!

File: README.md (hunk 1 above)
Comment (fcostaprojects) [reply README.md]: About the text doc think about the following situation:

with outputPath:
ImageResizer.createResizedImage(uri, width, height, format, quality, rotation, outputPath, outputFileName )

without outputPath
ImageResizer.createResizedImage(uri, width, height, format, quality, rotation, null, outputFileName )

But your text got more objective, I'm going to change

**Comment by fcostaprojects** [comment]
Ok, outputFilename better

**Comment by fcostaprojects** [comment]
Hi @4ian, how to resolve conflict in file index.d.ts ?

**Comment by Faolain** [comment]
@fcostaprojects any updates on this PR? seems like it would solve this issue https://github.com/bamlab/react-native-image-resizer/issues/77

**Comment by fcostaprojects** [comment]
@Faolain, I'm waiting for the resolution of conflicts. But it's working correctly. Solve #77
Test directly:

 "dependencies": {
    "react-native-image-resizer": "git+https://github.com/fcostaprojects/react-native-image-resizer.git",
   .....
}

**Comment by Faolain** [comment]
@4ian any suggestions on how fcostaprojects can resolve his typescript/flow conflicts?

**Pull request closed without merging**